study.md

```diff
@@ -22,4 +22,14 @@ should have something to contribute.
 
 ## Summary
 
-<!-- your notes here -->
+Relational databases are appropriate when the data you are storing is sensitive or relavant to the user storing the information where non-realational databases are more appropiate for store mass amounts of data that is used by many users as it is easier to scale.
```

Comment:
The big takeaway is that relational databases are always better if there are any relationships involved in the data.